components/layout/ImageContainer.tsx

```diff
@@ -6,6 +6,9 @@ interface ImageProps extends NextImageProps {
    * If true, the image will use the dimensions of the image instead of the dimensions passed in
    */
   useImageDimensions?: boolean;
+
+  // optional boolean, set to true if image is Largest Contentful Paint (LCP)
```

Comment:
Will you please update the comment to be the style in useImageDimensions above? If you hover over useImageDimensions anywhere in the code you see the description written! So we can use this for priority as well that way whenever someone is creating or updating an ImageContainer, they can easily get this information.

Reply:
@e-papanicolas 
oh that's so cool! Comment has been updated, and I can confirm it appears when you hover over the priority prop now.